packages/components/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Internal
+
+-   Replaced `ButtonGroup` with `ToggleGroupControl` component for "AM/PM" selector in DateTime component ([#61562](https://github.com/WordPress/gutenberg/pull/61562)).
```

Comment:
```suggestion
-   `DateTimePicker`: Replaced `ButtonGroup` with `ToggleGroupControl` component for "AM/PM" selector ([#61562](https://github.com/WordPress/gutenberg/pull/61562)).
```

Just a small tweak for easier scanning. Also, I think we can classify this as an Enhancement? Or maybe even a Bug Fix, since this fixes an accessibility bug. No strong opinion though.

Reply:
Moved to Enhancements, as I feel it enhances the accessibility experience.